spec.html

```diff
@@ -2676,8 +2676,10 @@ <h1>The <dfn variants="is an Object,is not an Object">Object Type</dfn></h1>
           An <dfn variants="accessor properties">accessor property</dfn> associates a key value with one or two accessor functions, and a set of Boolean attributes. The accessor functions are used to store or retrieve an ECMAScript language value that is associated with the property.
         </li>
       </ul>
-      <p>Properties are identified using key values. A <dfn variants="property keys">property key</dfn> value is either an ECMAScript String value or a Symbol value. All String and Symbol values, including the empty String, are valid as property keys. A <dfn id="property-name">property name</dfn> is a property key that is a String value.</p>
-      <p>An <dfn id="integer-index" variants="integer indices">integer index</dfn> is a String-valued property key that is a canonical numeric string and whose numeric value is either *+0*<sub>𝔽</sub> or a positive integral Number ≤ 𝔽(2<sup>53</sup> - 1). An <dfn id="array-index" variants="array indices">array index</dfn> is an integer index whose numeric value _i_ is in the range <emu-eqn>*+0*<sub>𝔽</sub> ≤ _i_ &lt; 𝔽(2<sup>32</sup> - 1)</emu-eqn>.</p>
+      <p>The properties of an object are uniquely identified using property keys. A <dfn variants="property keys">property key</dfn> is either a String or a Symbol. All Strings and Symbols, including the empty String, are valid as property keys. A <dfn id="property-name">property name</dfn> is a property key that is a String. An <dfn id="integer-index" variants="integer indices">integer index</dfn> is a property name _n_ such that CanonicalNumericIndexString(_n_) returns an integral Number in the inclusive interval from *+0*<sub>𝔽</sub> to 𝔽(2<sup>53</sup> - 1). An <dfn id="array-index" variants="array indices">array index</dfn> is an integer index _n_ such that CanonicalNumericIndexString(_n_) returns an integral Number in the inclusive interval from *+0*<sub>𝔽</sub> to 𝔽(2<sup>32</sup> - 2).</p>
```

Comment:
Why combine 2 paragraphs into a single paragraph? I have slightly preference for keeping the 2 paragraphs.

Reply:
Basically because there seems to be one topic: _property key_ and its subcategories. A paragraph boundary between defining _property name_ as a string-valued property key and _integer index_ as a property name canonically expressing a number in [+0, 2**53 - 1] strikes me as just a bit too artificial:

> The properties of an object are uniquely identified using property keys. A _property key_ is either a String or a Symbol. All Strings and Symbols, including the empty String, are valid as property keys. A _property name_ is a property key that is a String.
> 
> An _integer index_ is a property name <tt>n</tt> such that CanonicalNumericIndexString(<tt>n</tt>) returns an integral Number in the inclusive interval from **+0**<sub>𝔽</sub> to 𝔽(2<sup>53</sup> - 1). An _array index_ is an integer index <tt>n</tt> such that CanonicalNumericIndexString(<tt>n</tt>) returns an integral Number in the inclusive interval from **+0**<sub>𝔽</sub> to 𝔽(2<sup>32</sup> - 2).

But I don't feel too strongly about it.

Reply:
I'd prefer to keep the 2 paragraphs then.

Reply:
Done.